qanary-component-MT-Python-Helsinki-NLP/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  mt-helsinki-nlp:
+    # image: qanary/qanary-component-mt-python-helsinki-nlp:0.1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    network_mode: host
+    # ports:
+      # - 40120:40120
+    restart: unless-stopped
+    environment:
```

Comment:
@Perevalov I think it is not useful to define in a public GitHub-hosted file internal configuration.

Reply:
@anbo-de Changed in: https://github.com/WDAqua/Qanary-question-answering-components/pull/143/commits/a7c0c9970c1681511eb67aa716321edee8f0364d